diff --git a/rcu.py b/rcu.py
new file mode 100644
index 0000000..8ffaefd
--- /dev/null
+++ b/rcu.py
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+from src.main import main
+import os
+
+if __name__ == "__main__":
+   os.chdir(os.path.join(os.path.dirname(os.path.realpath(__file__)), "src"))
+   main()
diff --git a/src/__init__.py b/src/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/controllers/ConnectionDialogController.py b/src/controllers/ConnectionDialogController.py
index bae2db4..21eb24e 100644
--- a/src/controllers/ConnectionDialogController.py
+++ b/src/controllers/ConnectionDialogController.py
@@ -21,11 +21,11 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
 import socket
-from worker import Worker
+from ..worker import Worker
 import sys
 import pathlib
-import controllers
-import log
+from .. import controllers
+from .. import log
 from PySide2.QtCore import Qt, QSettings, QObject, QEvent, QPoint, \
     QCoreApplication
 from PySide2.QtWidgets import QMenu, QShortcut, QApplication, QAction, \
@@ -50,14 +50,14 @@ class ReturnKeyEventFilter(QObject):
             event.key() == Qt.Key_Enter):
             self.parent().menu().defaultAction().trigger()
         return QObject.eventFilter(self, obj, event)
-        
+
 
 
 
 class Preset:
     def __init__(self, manager, adict=None):
         self.manager = manager
-        
+
         if adict:
             self.name = adict['name']
             self.host = adict['host']
@@ -87,7 +87,7 @@ class Preset:
         # UI default state
         self.action.setCheckable(True)
         self.action.setChecked(False)
-        
+
     def clicked(self):
         self.manager.switch_to(self)
 
@@ -109,7 +109,7 @@ class Preset:
         # GUI things for the menu
         self.action.setChecked(False)
         self.action.setEnabled(True)
-                
+
 class PresetManager:
     def __init__(self, hostmenu, hostline, userline, passline):
         self.hostmenu = hostmenu
@@ -130,7 +130,7 @@ class PresetManager:
                 p.action)
         self.presets.append(p)
         return p
-    
+
     def switch_to(self, presetb):
         # switches to presetb
         for p in self.presets:
@@ -192,7 +192,7 @@ class PresetManager:
             password = None
             if QSettings().value('connection/pass'):
                 password = QSettings().value('connection/pass')
-            
+
             log.info('no presets exist; migrating from old config')
             settings = {'active': 0, 'presets': [{
                 'name': 'Preset 1',
@@ -225,10 +225,10 @@ class PresetManager:
                 sav['active'] = i
         QSettings().setValue('connection/presets', sav)
 
-    
+
 class ConnectionDialogController(controllers.UIController):
-    ui_filename = pathlib.Path('views', 'ConnectionDialog.ui')
-    
+    ui_filename = pathlib.Path('..', 'views', 'ConnectionDialog.ui')
+
     def __init__(self, model, threadpool):
         super(type(self), self).__init__(model, threadpool)
 
@@ -260,7 +260,7 @@ class ConnectionDialogController(controllers.UIController):
         save_btn.setMenu(self.preset_menu)
         save_btn.installEventFilter(
             OptionButtonEventFilter(self.window))
-        
+
 
         # For these, it would be better to search the QWidget for the
         # default QPushButton. Shortcut: just use connect_btn
@@ -295,7 +295,7 @@ class ConnectionDialogController(controllers.UIController):
         else:
             self.window.pass_lineEdit.setEchoMode(
                 QLineEdit.Password)
-        
+
 
     def add_new_preset(self):
         # This will add a new preset in the manager, and also refresh
@@ -331,7 +331,7 @@ class ConnectionDialogController(controllers.UIController):
 
     def get_preset_menu(self):
         menu = QMenu()
-        
+
         # This loads menu entries for presets
         self.presets = PresetManager(menu,
                                      self.window.host_lineEdit,
@@ -339,13 +339,13 @@ class ConnectionDialogController(controllers.UIController):
                                      self.window.pass_lineEdit)
         new = menu.addAction('Add new')
         new.triggered.connect(self.add_new_preset)
-        
+
         menu.addSeparator()
         rename = menu.addAction('Rename')
         rename.triggered.connect(self.rename_current_preset)
         save = menu.addAction('Save')
         save.triggered.connect(self.save_config)
-        
+
         menu.addSeparator()
         delete = menu.addAction('Delete')
         delete.triggered.connect(self.delete_current_preset)
@@ -370,7 +370,7 @@ class ConnectionDialogController(controllers.UIController):
         controllers.RecoveryOSController(self.model, skip_windows=True)\
                    .enter_recovery_mode(self.continue_loading,
                                         load_info=True)
-        
+
     def fill_config(self):
         # We want to fill the config prior to saving, but sometimes we
         # want to make a connection without saving. This will just fill
@@ -378,12 +378,12 @@ class ConnectionDialogController(controllers.UIController):
         self.model.config.host = self.window.host_lineEdit.text()
         self.model.config.user = self.window.user_lineEdit.text()
         self.model.config.password = self.window.pass_lineEdit.text()
-        
+
     def save_config(self):
         # Saves the current preset to settings file.
         self.fill_config()
         self.presets.save_current_preset()
-        
+
     def make_connection_async(self):
         log.info('make_connection_async')
         self.fill_config()
@@ -401,7 +401,7 @@ class ConnectionDialogController(controllers.UIController):
         self.window.save_pushButton.setEnabled(False)
         self.window.showpass_checkBox.setEnabled(False)
         self.cpbpretext = self.window.connect_pushButton.text()
-        
+
     def continue_loading(self, success=False):
         # Success is disregarded, check for ourselves (it was just
         # for compatibility)
diff --git a/src/controllers/ConnectionUtilityController.py b/src/controllers/ConnectionUtilityController.py
index 24bfca4..16f9ea7 100644
--- a/src/controllers/ConnectionUtilityController.py
+++ b/src/controllers/ConnectionUtilityController.py
@@ -23,10 +23,10 @@ from PySide2.QtCore import Qt, QObject, QEvent, QCoreApplication, \
     QSettings, QTimer
 from PySide2.QtWidgets import QListWidgetItem, QShortcut, QApplication
 from PySide2.QtGui import QKeySequence, QPalette, QPixmap
-from controllers import UIController
-from panes import paneslist, IncompatiblePane
+from ..controllers import UIController
+from ..panes import paneslist, IncompatiblePane
 from pathlib import Path
-import log
+from .. import log
 
 PANEROLE = 420
 
@@ -146,7 +146,7 @@ QListWidget::item::selected  {
         quitshortcut.activated.connect(QApplication.instance().quit)
         closeshortcut = QShortcut(QKeySequence.Close, self.window)
         closeshortcut.activated.connect(QApplication.instance().quit)
-        
+
         self.window.show()
         self.cx_timer.start()
 
@@ -183,7 +183,7 @@ QListWidget::item::selected  {
             previous_pane = self.current_pane.data(PANEROLE)
             layout.removeWidget(previous_pane.window)
             previous_pane.window.hide()
-        
+
         # load new pane into frame
         layout.addWidget(new_pane.window)
         new_pane.window.show()
@@ -208,7 +208,7 @@ QListWidget::item::selected  {
             newitem = QListWidgetItem(
                 pane.get_icon(),
                 pane.name)
-            
+
             # Check compatibility
             no_compat_check = QCoreApplication.args.no_compat_check
             if no_compat_check:
diff --git a/src/controllers/RecoveryOSController.py b/src/controllers/RecoveryOSController.py
index 3afac5c..0e38349 100644
--- a/src/controllers/RecoveryOSController.py
+++ b/src/controllers/RecoveryOSController.py
@@ -22,7 +22,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from . import UIController
 from pathlib import Path
-import log
+from .. import log
 from PySide2.QtCore import QTimer
 
 class RecoveryOSController(UIController):
@@ -85,6 +85,3 @@ class RecoveryOSController(UIController):
             QTimer.singleShot(30000, lambda: self.leave_stage_1(endcb))
         else:
             endcb(self.model.is_in_recovery)
-
-            
-        
diff --git a/src/controllers/UIController.py b/src/controllers/UIController.py
index 1409f82..81760a2 100644
--- a/src/controllers/UIController.py
+++ b/src/controllers/UIController.py
@@ -23,16 +23,16 @@ import pathlib
 import sys
 from PySide2.QtCore import QFile, QCoreApplication
 from PySide2.QtUiTools import QUiLoader
-import log
+from .. import log
 import re
 
 class UIController:
     identity = ''
     name = ''
     ui_filename = ''
-    # pyinstaller moves all datafiles into a tmp dir. This basepath
+    # This basepath
     # can be overridden by a subclass, like third-party panes.
-    ui_basepath = '.'
+    ui_basepath = pathlib.Path(__file__).parent
     if hasattr(sys, '_MEIPASS'):
         ui_basepath = sys._MEIPASS
 
@@ -107,7 +107,7 @@ class UIController:
             return False
 
         return True
-    
+
     def __init__(self, model, threadpool=None):
         self.model = model
         self.threadpool = threadpool
diff --git a/src/main.py b/src/main.py
index a8b2b3a..56a76ec 100644
--- a/src/main.py
+++ b/src/main.py
@@ -37,16 +37,13 @@ if 'Darwin' == platform.system():
     os.environ['LANG'] = 'en_US.UTF-8'
 
 # Give these to all our children
-global worker
-import worker
-global log
-import log
-global svgtools
-import svgtools as svgtools
+from . import worker
+from . import log
+from . import svgtools
 
-import model
-from controllers import ConnectionDialogController
-from panes import paneslist
+from . import model
+from .controllers import ConnectionDialogController
+from .panes import paneslist
 
 
 from pathlib import Path
@@ -109,7 +106,7 @@ if args.cli:
 
 
 # Start main application
-if __name__ == '__main__':
+def main():
     QCoreApplication.setAttribute(Qt.AA_DisableWindowContextHelpButton)
     QCoreApplication.setAttribute(Qt.AA_ShareOpenGLContexts)
     QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling)
@@ -122,7 +119,7 @@ if __name__ == '__main__':
     QCoreApplication.args = args
 
     # Version is now stored in version.txt
-    ui_basepath = '.'
+    ui_basepath = Path(__file__).parent
     if hasattr(sys, '_MEIPASS'):
         ui_basepath = sys._MEIPASS
     versiontxt = Path(Path(ui_basepath) / 'version.txt')
@@ -162,7 +159,7 @@ if __name__ == '__main__':
         QCoreApplication.sharePath = share_dir
         QSettings().setValue('main/share_path', str(share_dir))
     QCoreApplication.sharePath.mkdir(parents=True, exist_ok=True)
-    
+
     app = QApplication(sys.argv)
 
     # Keep consistent style across platforms (hard to write code when
@@ -214,7 +211,7 @@ if __name__ == '__main__':
     app.setPalette(palette)
 
     # The model and threadpool are distributed to all panes.
-    model = model.RCU(QCoreApplication)
+    mdl = model.RCU(QCoreApplication)
     threadpool = QThreadPool()
 
     # Skip main application for rendering RMN to PDF. Use dummy docs.
@@ -232,6 +229,5 @@ if __name__ == '__main__':
             sys.exit(1)
 
     # Start the main application with the Connection Dialog.
-    connection_dialog = ConnectionDialogController(model,
-                                                   threadpool)
+    connection_dialog = ConnectionDialogController(mdl, threadpool)
     sys.exit(app.exec_())
diff --git a/src/model/battery.py b/src/model/battery.py
index 9894cca..55538a2 100644
--- a/src/model/battery.py
+++ b/src/model/battery.py
@@ -19,7 +19,7 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-import log
+from .. import log
 import time
 
 class Battery:
diff --git a/src/model/collection.py b/src/model/collection.py
index c99ea21..9e39d56 100644
--- a/src/model/collection.py
+++ b/src/model/collection.py
@@ -19,7 +19,7 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-import log
+from .. import log
 from datetime import datetime
 import uuid
 import json
diff --git a/src/model/config.py b/src/model/config.py
index 5de03d6..9933c9b 100644
--- a/src/model/config.py
+++ b/src/model/config.py
@@ -21,7 +21,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
 import paramiko
-import log
+from .. import log
 from pathlib import Path
 import json
 from .transport import Transport
diff --git a/src/model/display.py b/src/model/display.py
index e10e008..47f2edf 100644
--- a/src/model/display.py
+++ b/src/model/display.py
@@ -44,7 +44,7 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 '''
 
-import log
+from .. import log
 from PySide2.QtCore import QByteArray, QBuffer, QIODevice
 from PySide2.QtGui import QImage, QMatrix
 import math
diff --git a/src/model/document.py b/src/model/document.py
index bdc8315..1701180 100644
--- a/src/model/document.py
+++ b/src/model/document.py
@@ -20,15 +20,15 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-import log
+from .. import log
 import tempfile
 from datetime import datetime
 import tarfile
 from pathlib import Path
-from model.template import Template
+from ..model.template import Template
 import uuid
 import json
-from model import lines
+from ..model import lines
 import gc
 import shutil
 import time
@@ -43,10 +43,10 @@ from PySide2.QtCore import Qt, QByteArray, QIODevice, QBuffer, QSizeF, \
     QSettings, QRectF, QPointF
 from PySide2.QtPrintSupport import QPrinter
 
-from pdfrw.pdfrw import PdfReader, PdfWriter, PageMerge, PdfDict, \
+from pdfrw import PdfReader, PdfWriter, PageMerge, PdfDict, \
     PdfArray, PdfName, IndirectPdfDict, uncompress, compress
 
-import svgtools
+from .. import svgtools
 
 def rmdir(path):
     if path.is_file() and path.exists():
@@ -1508,7 +1508,7 @@ class DocumentPage:
             layer.render_to_painter(painter, vector)
 
             
-from model.pens import *
+from ..model.pens import *
 class DocumentPageLayer:
     pen_widths = []
 
diff --git a/src/model/rcu.py b/src/model/rcu.py
index 9d07733..12dfd69 100644
--- a/src/model/rcu.py
+++ b/src/model/rcu.py
@@ -20,11 +20,11 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
 from .config import Config
-import log
+from .. import log
 
 from PySide2.QtCore import QThreadPool, QSettings, QTimer
 from PySide2.QtGui import QImage
-from worker import Worker
+from ..worker import Worker
 import select
 import base64
 from .template import Template
diff --git a/src/model/template.py b/src/model/template.py
index b31529b..b5775ab 100644
--- a/src/model/template.py
+++ b/src/model/template.py
@@ -21,15 +21,15 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 # These need to be cleaned up -- not all these imports are required
 from pathlib import Path
-import log
+from .. import log
 import json
 from PySide2.QtCore import QByteArray
-from worker import Worker
+from ..worker import Worker
 import tempfile
 import uuid
 import tarfile
 import zipfile
-import svgtools
+from .. import svgtools
 import os
 
 
diff --git a/src/panes/about/__init__.py b/src/panes/about/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/about/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/about/pane.py b/src/panes/about/pane.py
index 173ed2b..55e888f 100644
--- a/src/panes/about/pane.py
+++ b/src/panes/about/pane.py
@@ -19,12 +19,12 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-import log
+from ... import log
 from pathlib import Path
-from controllers import UIController
+from ...controllers import UIController
 import sys
 
-from worker import Worker
+from ...worker import Worker
 
 from PySide2.QtCore import QByteArray, QUrl, QSize, QCoreApplication
 from PySide2.QtGui import QIcon
diff --git a/src/panes/deviceinfo/BatteryInfoController.py b/src/panes/deviceinfo/BatteryInfoController.py
index 6fc1ca5..fbda766 100644
--- a/src/panes/deviceinfo/BatteryInfoController.py
+++ b/src/panes/deviceinfo/BatteryInfoController.py
@@ -19,9 +19,9 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-import log
+from ... import log
 
-from controllers import UIController
+from ...controllers import UIController
 from pathlib import Path
 
 class BatteryInfoController(UIController):
diff --git a/src/panes/deviceinfo/__init__.py b/src/panes/deviceinfo/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/deviceinfo/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/deviceinfo/backup/Backup.py b/src/panes/deviceinfo/backup/Backup.py
index e3707a4..8c4aa26 100644
--- a/src/panes/deviceinfo/backup/Backup.py
+++ b/src/panes/deviceinfo/backup/Backup.py
@@ -25,7 +25,7 @@ import uuid
 import datetime
 import json
 from pathlib import Path
-import log
+from .... import log
 import time
 import platform
 import base64
diff --git a/src/panes/deviceinfo/backup/BackupController.py b/src/panes/deviceinfo/backup/BackupController.py
index cd70dc9..85c7164 100644
--- a/src/panes/deviceinfo/backup/BackupController.py
+++ b/src/panes/deviceinfo/backup/BackupController.py
@@ -23,7 +23,7 @@ import tarfile
 import tempfile
 import pathlib
 from pathlib import Path
-import log
+from .... import log
 from .Backup import Backup
 import json
 from datetime import datetime
diff --git a/src/panes/deviceinfo/backup/BackupFile.py b/src/panes/deviceinfo/backup/BackupFile.py
index 97d395a..b011224 100644
--- a/src/panes/deviceinfo/backup/BackupFile.py
+++ b/src/panes/deviceinfo/backup/BackupFile.py
@@ -23,7 +23,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import hashlib
 from pathlib import Path
-import log
+from .... import log
 
 class BackupFile:
     # This is a single file contined inside of a backup.
diff --git a/src/panes/deviceinfo/pane.py b/src/panes/deviceinfo/pane.py
index 1c9a42b..ae838c9 100644
--- a/src/panes/deviceinfo/pane.py
+++ b/src/panes/deviceinfo/pane.py
@@ -21,9 +21,9 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from pathlib import Path
 import math
-import controllers
-import log
-from worker import Worker
+from ... import controllers
+from ... import log
+from ...worker import Worker
 from . import backup
 from PySide2.QtWidgets import QInputDialog, QLineEdit, QMessageBox
 from PySide2.QtGui import QIcon, QPixmap
diff --git a/src/panes/display/__init__.py b/src/panes/display/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/display/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/display/pane.py b/src/panes/display/pane.py
index 89f657c..d931813 100644
--- a/src/panes/display/pane.py
+++ b/src/panes/display/pane.py
@@ -25,9 +25,9 @@ from PySide2.QtGui import QImage, QPixmap, QColor, QIcon
 from PySide2.QtWidgets import QFileDialog, QWidget
 from pathlib import Path
 from datetime import datetime
-from controllers import UIController
-from worker import Worker
-import log
+from ...controllers import UIController
+from ...worker import Worker
+from ... import log
 import gc
 import ctypes
 
diff --git a/src/panes/incompatible/__init__.py b/src/panes/incompatible/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/incompatible/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/incompatible/pane.py b/src/panes/incompatible/pane.py
index 53be52c..ffea41b 100644
--- a/src/panes/incompatible/pane.py
+++ b/src/panes/incompatible/pane.py
@@ -21,8 +21,8 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
 from pathlib import Path
-from controllers import UIController
-import log
+from ...controllers import UIController
+from ... import log
 from PySide2.QtGui import QPixmap
 
 class IncompatiblePane(UIController):
diff --git a/src/panes/notebooks/__init__.py b/src/panes/notebooks/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/notebooks/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/notebooks/pane.py b/src/panes/notebooks/pane.py
index 7320790..1517483 100644
--- a/src/panes/notebooks/pane.py
+++ b/src/panes/notebooks/pane.py
@@ -19,12 +19,12 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-from controllers import UIController
+from ...controllers import UIController
 from pathlib import Path
-import log
-from model.document import Document
-from model.collection import Collection
-from worker import Worker
+from ... import log
+from ...model.document import Document
+from ...model.collection import Collection
+from ...worker import Worker
 import json
 from datetime import datetime
 import platform
diff --git a/src/panes/software/__init__.py b/src/panes/software/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/software/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/software/pane.py b/src/panes/software/pane.py
index 468aff0..1057ea7 100644
--- a/src/panes/software/pane.py
+++ b/src/panes/software/pane.py
@@ -19,8 +19,8 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-import log
-from controllers import UIController
+from ... import log
+from ...controllers import UIController
 from pathlib import Path
 from PySide2.QtCore import Qt, QSize, QRect, QSettings
 from PySide2.QtGui import QIcon
diff --git a/src/panes/splash/__init__.py b/src/panes/splash/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/splash/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/splash/pane.py b/src/panes/splash/pane.py
index 6ca58c8..aed8f84 100644
--- a/src/panes/splash/pane.py
+++ b/src/panes/splash/pane.py
@@ -24,8 +24,8 @@ from PySide2.QtGui import QPixmap, QImage, QIcon
 from PySide2.QtWidgets import QFileDialog
 from pathlib import Path
 from datetime import datetime
-import log
-from controllers import UIController
+from ... import log
+from ...controllers import UIController
 
 class ResizeEventFilter(QObject):
     def eventFilter(self, obj, event):
diff --git a/src/panes/templates/__init__.py b/src/panes/templates/__init__.py
new file mode 100644
index 0000000..143f486
--- /dev/null
+++ b/src/panes/templates/__init__.py
@@ -0,0 +1 @@
+# __init__.py
diff --git a/src/panes/templates/importcontroller.py b/src/panes/templates/importcontroller.py
index 296229c..0fad2ce 100644
--- a/src/panes/templates/importcontroller.py
+++ b/src/panes/templates/importcontroller.py
@@ -20,9 +20,9 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-from controllers import UIController
+from ...controllers import UIController
 from pathlib import Path
-import log
+from ... import log
 
 from PySide2.QtCore import QByteArray, QSize, Qt, QTimer, QBuffer, \
     QIODevice, QCoreApplication
@@ -36,10 +36,10 @@ import os
 import io
 import tempfile
 
-from worker import Worker
-import svgtools
+from ...worker import Worker
+from ... import svgtools
 
-from model.template import Template
+from ...model.template import Template
 
 
 def get_square_box(box):
diff --git a/src/panes/templates/pane.py b/src/panes/templates/pane.py
index 4784a5e..93e20e6 100644
--- a/src/panes/templates/pane.py
+++ b/src/panes/templates/pane.py
@@ -19,9 +19,9 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 '''
 
-from controllers import UIController
+from ...controllers import UIController
 from pathlib import Path
-import log
+from ... import log
 from PySide2.QtWidgets import QTreeWidgetItem, QTreeWidget, QMenu, \
     QFrame, QAbstractItemView, QMessageBox, QHeaderView, QFileDialog, \
     QSizePolicy
@@ -29,11 +29,11 @@ from PySide2.QtSvg import QSvgWidget
 from PySide2.QtCore import Qt, QRect, QByteArray, QSize, QObject, \
     QEvent, QSettings, QCoreApplication
 from PySide2.QtGui import QIcon
-from model.template import Template
+from ...model.template import Template
 import platform
 import re
 
-import svgtools
+from ... import svgtools
 
 from .importcontroller import TemplateImporter
 
diff --git a/src/svgtools.py b/src/svgtools.py
index 88a0c00..378a534 100644
--- a/src/svgtools.py
+++ b/src/svgtools.py
@@ -23,7 +23,7 @@ from PySide2.QtCore import QByteArray, QIODevice, QBuffer
 from PySide2.QtGui import QImage, QPainter, QPixmap
 from PySide2.QtSvg import QSvgRenderer
 
-import log
+from . import log
 
 import os
 
